course/tests.py

```diff
@@ -1,3 +1,29 @@
+from django.core.management import CommandError, find_commands
 from django.test import TestCase
+from django.core import management
+from io import StringIO
+import sys
 
-# Create your tests here.
+
+class CommandTests(TestCase):
+    databases = '__all__'
+    fixtures = ["dumps/courseData.json"]
+
+    def test_command(self):
+        sys.stdout = out = StringIO()
+        management.call_command('exportcourse', 1, stdout=out, verbosity=3)
+        self.assertIn("Exporting course Spanish for English speakers course", out.getvalue())
+
+    def test_calling_a_command_without_argument(self):
+        with self.assertRaisesMessage(CommandError, 'Error: the following arguments are required: course_id'):
+            sys.stdout = out = StringIO()
+            management.call_command('exportcourse', stdout=out)
+
+    def test_exportcourses(self):
+        with self.assertRaisesMessage(CommandError, "Unknown command: 'exportcourses'"):
+            management.call_command(('exportcourses',))
+
+    def test_check_course_does_not_exists(self):
```

Comment:
This test case looks good! But maybe use a diffferent id, because 3 might exist in the future. I'm pretty sure something like 345345 would never exist (and if it ever gets to that, we'll have a real text fixture here instead of the actual data anyway)

Reply:
Okay, will add large number.